app/controllers/admin/dashboard_controller.rb

```diff
@@ -9,9 +9,9 @@ class Admin::DashboardController < Admin::BaseController
   def index
     #@topics = Topic.mine(current_user.id).pending.page params[:page]
 
-    if current_user.is_admin? || current_user.is_agent?
+    if (current_user.is_admin? or current_user.is_agent?) and (forums? or tickets?)
```

Comment:
The [Ruby Style Guide](https://github.com/bbatsov/ruby-style-guide#no-and-or-or) recommends using `||` over `or`